astropy/__init__.py

```diff
@@ -6,36 +6,16 @@
 managing them.
 """
 
-import os
+import pathlib
 import sys
 
 from .version import version as __version__
 
-
-def _is_astropy_source(path=None):
-    """
-    Returns whether the source for this module is directly in an astropy
-    source distribution or checkout.
-    """
-
-    # If this __init__.py file is in ./astropy/ then import is within a source
-    # dir .astropy-root is a file distributed with the source, but that should
-    # not installed
-    if path is None:
-        path = os.path.join(os.path.dirname(__file__), os.pardir)
-    elif os.path.isfile(path):
-        path = os.path.dirname(path)
-
-    source_dir = os.path.abspath(path)
-    return os.path.exists(os.path.join(source_dir, ".astropy-root"))
-
-
 # The location of the online documentation for astropy
 # This location will normally point to the current released version of astropy
-if "dev" in __version__:
-    online_docs_root = "https://docs.astropy.org/en/latest/"
-else:
-    online_docs_root = f"https://docs.astropy.org/en/{__version__}/"
+online_docs_root = "https://docs.astropy.org/en/{}/".format(
+    "latest" if "dev" in __version__ else f"v{__version__}"
+)
```

Comment:
```suggestion
online_docs_root = f"https://docs.astropy.org/en/{'latest' if 'dev' in __version__ else f'v{__version__}'}/"
```

Reply:
I do think the `.format` version is more readable in this case.

Reply:
> ```suggestion
> 
> online_docs_root = f"https://docs.astropy.org/en/{'latest' if 'dev' in __version__ else f'v{__version__}'}/"
> 
> ```

This requires a noqa f501

Reply:
```suggestion
online_docs_root = (
    "https://docs.astropy.org/en/"
    f"{'latest' if 'dev' in __version__ else f'v{__version__}'}/"
)
```

Reply:
If we go that route I'd suggest
```
online_docs_root = (
    "https://docs.astropy.org/en/"
    + ("latest/" if "dev" in __version__ else f"v{__version__}/")
)
```
Though I still think `.format()` is better in this case :smiley_cat: